engine/src/main/resources/org/camunda/bpm/engine/db/liquibase/baseline/liquibase.mariadb.create.engine.sql

```diff
@@ -201,6 +201,7 @@ create table ACT_RU_TASK (
     FOLLOW_UP_DATE_ datetime(3),
     SUSPENSION_STATE_ integer,
     TENANT_ID_ varchar(64),
+    TASK_STATE_ varchar(64),
```

Comment:
❌ Unnecessary:
```suggestion
```

Reply:
Done. Please review the changes in latest pushed code